backend/src/v5/schemas/tickets/templates.js

```diff
@@ -153,9 +159,13 @@ const configSchema = Yup.object().shape({
 	defaultView: defaultFalse,
 	defaultImage: Yup.boolean().when('defaultView', (defaultView, schema) => (defaultView ? schema.strip() : defaultFalse)),
 	pin: Yup.lazy((val) => (val?.color ? Yup.object({ color: pinColSchema }) : defaultFalse)),
+	status: Yup.lazy((val) => (val ? Yup.object({
```

Comment:
why do we need lazy here?

Reply:
we do that for pin so I think we have to do something similar here 🤔

Reply:
that's because a pin could either be an object or a boolean (due to old schema)

status either is an object or it doesn't exist so I don't think we need a lazy evaluation here...